app/Http/Controllers/Api/V1/SearchController.php

```diff
@@ -175,6 +176,99 @@ public function search(SearchRequest $searchRequest)
+     * @queryParam  isPublic Public access enabled or disabled. Example: true
+     * @queryParam  elasticsearch Elasticsearch enabled or disabled. Example: false
+     * @queryParam  startDate Start date for search by date range. Example: 2020-04-30 00:00:00
+     * @queryParam  endDate End date for search by date range. Example: 2020-04-30 23:59:59
+     * @queryParam  h5pLibraries Array of h5p libraries to match. Example: ['H5P.InteractiveVideo 1.21']
+     * @queryParam  model Index to filter by. Example: activities
+     * @queryParam  sort Field to sort by. Example: created_at
+     * @queryParam  order Order to sort by. Example: desc
+     * @queryParam  from Index where the pagination start from. Example: 0
+     * @queryParam  size Number of records to return. Example: 10
+     *
+     * @apiResourceCollection  App\Http\Resources\V1\SearchResource
+     * @apiResourceModel  App\Models\Activity
+     *
+     * @response  {
+     *     "data": [
+     *         {
+     *             "id": 457,
+     *             "thumb_url": "/storage/uploads/3zjZuLoQrRk0MZ5wP7UPyOut5zUybf3tW3a4Q2M1.png",
+     *             "title": "Text Structure Lesson 2 Problem and Solution",
+     *             "description": "Learning Objective\nThe learner will define and describe the main characteristics...",
+     *             "model": "Project",
+     *             "user": {
+     *                 "id": 1,
+     *                 "name": "localuser",
+     *                 "email": "localuser@local.com",
+     *                 "email_verified_at": null,
+     *                 "created_at": "2020-08-22T12:13:52.000000Z",
+     *                 "updated_at": "2020-08-22T12:13:52.000000Z",
+     *                 "first_name": "test",
+     *                 "last_name": "test",
+     *                 "organization_name": "organization_name",
+     *                 "job_title": "job_title",
+     *                 "address": null,
+     *                 "phone_number": null,
+     *                 "organization_type": null,
+     *                 "website": null,
+     *                 "deleted_at": null,
+     *                 "role": null,
+     *                 "gapi_access_token": null,
+     *                 "pivot": {
+     *                     "project_id": 457,
+     *                     "user_id": 1,
+     *                     "role": "owner",
+     *                     "created_at": "2020-08-25T09:35:35.000000Z",
+     *                     "updated_at": "2020-08-25T09:35:35.000000Z"
+     *                 }
+     *             }
+     *         },
+     *         {
+     *             "id": 1102,
+     *             "title": "All About That Text",
+     *             "model": "Playlist",
+     *             "user": null
+     *         }
+     *     ],
+     *     "meta": {
+     *         "projects": 7,
+     *         "playlists": 3,
+     *         "activities": 2,
+     *         "total": 12
+     *     }
+     * }
```

Comment:
Please create response file like other document block.

Reply:
Also please confirm about the response format.

Reply:
Moved the updated API doc response format to separate files.